packages/rest-api-client/rollup.config.js

```diff
@@ -34,10 +34,10 @@ export default {
               // see kintone's supported browsers https://get.kintone.help/general/en/user/list_start/webbrowser.html
               browsers: [
                 "IE 11",
-                "last 2 edge versions",
-                "last 2 firefox version",
-                "last 2 chrome versions",
-                "last 2 safari versions",
+                "last 1 edge versions",
+                "last 1 firefox version",
+                "last 1 chrome versions",
+                "last 1 safari versions",
                 "iOS 11",
```

Comment:
```suggestion
                "iOS 12",
```

reference in [Kintone Supported Web Browsers](https://get.kintone.help/general/en/user/list_start/webbrowser.html)

Reply:
Thank you for your suggestion!
Because select the iOS browser version 12 directly, I fixed iOS browser version as `iOS >= 12`.